### PR TITLE
[RDCC-5611]Special Tribunals : Update JRD Service Code Mapping table for Special Tribunals

### PR DESCRIPTION
**Pull request by manukundloo-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-5611

### Change description ###

[RDCC-5611]Special Tribunals : Update JRD Service Code Mapping table for Special Tribunals

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V1_48__update_judicial_service_code_mapping.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO judicial_service_code_mapping (service_id,ticket_code,service_code,service_description)
+VALUES (117,'374','BCA1','Care Standards'),
+       (118,'374','BCA2','Mental Health'),
+       (119,'374','BCA3','Primary Health Lists'),
+       (120,'374','BCA4','Special Educational Needs'),
+       (121,'376','BBA2','Criminal Injuries Compensation');
+
+COMMIT;
```
Comment (sahitya-desireddy) [inline src/main/resources/db/migration/V1_48__update_judicial_service_code_mapping.sql:8]: Request @manukundloo-hmcts can we please verify the same scripts should be updated in integration test cases as well please :)

Comment (manukundloo-hmcts) [reply src/main/resources/db/migration/V1_48__update_judicial_service_code_mapping.sql]: Done the changes for Functional test cases and also wrote all DB change code in one file

**Review by vilasshelke-hmcts — APPROVED** [review]

**Comment by manukundloo-hmcts** [comment]
Closing the PR since it is failing with owasp issue.

**Comment by manukundloo-hmcts** [comment]
Reopening the PR

**Review by AdilOoze — APPROVED** [review]

**Review by vasthav-patta — APPROVED** [review]

**Pull request merged**